packages/graphql-playground-react/src/components/Playground.tsx

```diff
@@ -255,15 +255,21 @@ export class Playground extends React.PureComponent<Props & ReduxProps, State> {
     const props = propsInput || this.props
     const endpoint = props.sessionEndpoint || props.endpoint
     const currentSchema = this.state.schema
+    const globalHeaders = props.settings['request.globalHeaders']
+
     try {
       const data = {
         endpoint,
         headers:
           props.sessionHeaders && props.sessionHeaders.length > 0
-            ? props.sessionHeaders
-            : props.headers && Object.keys(props.headers).length > 0
-              ? JSON.stringify(props.headers)
-              : undefined,
```

Comment:
I'm not sure about this merge conflict.  Is it important that headers are `undefined` in this case when there are no `props.headers` ? @acao @yoshiakis

Reply:
Sorry for very late! No, `undefined` here isn't important any more after merging this PR. This `undefined` is for avoiding [this issue](#965) and the changes you've made avoid it too.